eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -105,7 +105,7 @@ $responses = @{}
 
 $packageProperties = Get-ChildItem -Recurse -Force "$configFileDir" `
   | Where-Object { 
-      $_.Extension -eq '.json' -and ($_.FullName.Substring($configFileDir.Length + 1) -notmatch '^_.*?\\')
+      $_.Extension -eq '.json' -and ($_.FullName.Substring($configFileDir.Length + 1) -notmatch '^_.*?[\\/]')
```

Comment:
What is this check actually looking for? Are you looking to see if the file name starts with _? If so isn't easier to just check `$_.FullName.StartsWith("_")`?

Reply:
The goal is to avoid looking in directories created automatically by the DevOps pipeline.  Specifically, the `_manifest` directory. Am trying to match against directories in the `ConfigFileDir` so I can't use FulName.

Reply:
Is the regex valid? I assume you need to use escape char for delimeter '/'